README.md

```diff
@@ -5,6 +5,15 @@ zathura is a highly customizable and functional document viewer based on the gir
 library and several document libraries. This plugin for zathura provides PDF support using the
 `poppler` rendering library.
 
+This fork adds signature validation support to the PDF plugin.
+
+As the signature stamp content (name of the signer, timestamp, and more) is defined by the signing application, this information can be altered at signature creation. It does not have to adhere to the actual cryptographic signing certificate subject.
+Therefore, this plugin validates the signature and hides the forgeable information by placing an overlay containing the trusted signing certificate subject on the signature stamp. Furthermore, the validation result is communicated to the user by coloring the overlay.
+
+![alt text](./pictures/overlays.png "Signature overlays")
+
+The plugin can be built with meson, and subsequently used in zathura by specifying the `-p builddir/` option.
+
```

Comment:
Not relevant for the zathura itself. overlays.png can also be dropped. There might be some value in documenting the feature, but this ideally would document that signature verification is supported and how it is displayed.